### PR TITLE
feat: add Animated Typings and fix error where pitch could not be set for android

### PR DESCRIPTION
**Pull request by Blackfaded**
This pr adds pitch support for android and adds typings for the animated namespace

### CODE REVIEW

**Comment by ferdicus** [comment]
Hey there, thanks for your contribution.

While you're at it, could you also clean up the JS side please?

I think, there is no need for passing `followUserPitch` here: https://github.com/react-native-mapbox-gl/maps/blob/a8b6d336981a79874470432db41b45be88a7fe19/javascript/components/Camera.js#L545

Also, would be nice if you could then extend the docs with a commend above the prop, here: https://github.com/react-native-mapbox-gl/maps/blob/a8b6d336981a79874470432db41b45be88a7fe19/javascript/components/Camera.js#L109

And finally, if you're feeling super adventurous, you could even add a little js test :)

Cheers

**Comment by Blackfaded** [comment]
> Hey there, thanks for your contribution.
> 
> While you're at it, could you also clean up the JS side please?
> 
> I think, there is no need for passing `followUserPitch` here:
> 
> https://github.com/react-native-mapbox-gl/maps/blob/a8b6d336981a79874470432db41b45be88a7fe19/javascript/components/Camera.js#L545
> 
> Also, would be nice if you could then extend the docs with a commend above the prop, here:
> 
> https://github.com/react-native-mapbox-gl/maps/blob/a8b6d336981a79874470432db41b45be88a7fe19/javascript/components/Camera.js#L109
> 
> And finally, if you're feeling super adventurous, you could even add a little js test :)
> 
> Cheers

Hey, of course!
I also updated the camera docs a little bit.

**Comment by ferdicus** [comment]
Great, thanks 👍

File: docs/Camera.md [hunk 1]
```diff
@@ -30,16 +30,16 @@
 | &nbsp;&nbsp;paddingTop | `number` | `none` | `false` | Top camera padding for bounds |
 | &nbsp;&nbsp;paddingBottom | `number` | `none` | `false` | Bottom camera padding for bounds |
 | zoomLevel | `number` | `none` | `false` | Zoom level of the map |
-| minZoomLevel | `number` | `none` | `false` | FIX ME NO DESCRIPTION |
-| maxZoomLevel | `number` | `none` | `false` | FIX ME NO DESCRIPTION |
+| minZoomLevel | `number` | `none` | `false` | The minimun zoom level of the map |
```
Comment (mfazekas) [inline docs/Camera.md:33]: These are autgenerated files, next time pls fix the in the source code Camera.js, not in Camera.md.

**Pull request merged**